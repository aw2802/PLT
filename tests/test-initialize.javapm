public class TestInitialize {
	public void main() {
		int x;
		print("It's okay not to initialize.");
	}

}
