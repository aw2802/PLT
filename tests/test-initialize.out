It's okay not to initialize.
